run.sh

```diff
@@ -3,74 +3,69 @@ until curl -s "http://127.0.0.1:9000/api/system/lbstatus"; do
   echo 'graylog not ready, sleeping for 3 seconds'
   sleep 3
 done
+printf "\n"
 
-graylog_api="http://admin:${ADMIN_PASSWORD}@127.0.0.1:9000/api"
+graylog_api="http://admin:$ADMIN_PASSWORD@127.0.0.1:9000/api"
```

Comment:
Since we don't need them, I thought to remove the clutter.

I looked in that guide and didn't find a reference to that. Now that I found it,  I'll put those back. I prefer to follow the style guide